### ISSUE TITLE
Errors: store (new) works, update (existing) does not

### ISSUE THREAD

**Issue by pspiteri** [dwightwatson/validating#36]
Sorry mate, I may have another one.

The 0.8 release works great with creating new records, but updating existing records ... I think this

```
if ( ! $post->save())
```

is working fine - it is not saving if validation fails. However, I think that when it fails a validation it is not sending back an error message.

To test this I used the code I in a store method I have in my controller, which worked fined (so my view should be ok), and pasted it into an update controller. I made one change (so it would find the right record, of course) and ... well ... no error message on Redirect::back().

Then I used some alternate validation I have in place (based on a Laracasts episode) and my validations and errors worked fine. I did this just to test my validators and they seemed to work just fine.

So I thought I should raise this with you. Model code below, and it was the unique validator in particular that I was testing to fail. Again this works fine for store (new), but not for update (existing).

Cheers

```
<?php

use Illuminate\Database\Eloquent\SoftDeletingTrait;
use Watson\Validating\ValidatingTrait;


class Client extends Eloquent {

    use SoftDeletingTrait;
    use ValidatingTrait;

    protected $throwValidationExceptions = false;
    protected $dates = ['deleted_at'];
    protected $fillable = ['name'];
    protected $rules = ['name' => ['required', 'unique:clients']];

    public function users()
    {
        return $this->hasMany('User');
    }

    public function contracts()
    {
        return $this->hasMany('Contract');
    }

}
```
